tests/integration/test_extension.py

```diff
@@ -69,6 +69,50 @@ def test_paging(self):
             privilege = self.app.phpbb3.fetch_acl_options(skip=skip, limit=1)
             self.assertEqual((skip, privilege), expected_privileges[skip])
 
+    def test_fetch_global_topics(self):
+        # type: () -> None
+        base._create_global_topics(self.cursor)
+
+        expected_topics = [(0, [{
+                                'topic_id': 0,
```

Comment:
Don't use only positive examples.

At a glance, we have here several things to watch out:
  * positive cases
  * negative cases
    * wrong type
    * not in requested forum (`WHERE` clause contains search over `forum_id`)

Reply:
Added a few negative tests. Wrong forum, not existent topic, wrong/different topic and different type of data in expected and read.